package.json

```diff
@@ -24,7 +24,7 @@
   "lint-staged": {
     "*": "prettier --write --ignore-unknown",
     "packages/**/*.{ts,tsx,js,jsx}": "eslint -c packages/.eslintrc.json --fix",
-    "packages/maskbook/**/*": "gulp locale-kit --sync-keys --remove-unused-keys"
+    "packages/maskbook/**/*": "sh -c 'gulp locale-kit --sync-keys --remove-unused-keys'"
```

Comment:
macOS and Linux workaround

Windows: no idea

Reply:
CC @Jack-Works

Reply:
Why do you need a `sh`?

Reply:
you need gulp allow input any parameter

see https://github.com/okonet/lint-staged#packagejson-example

Reply:
Maybe you can write a small `.js` file to execute that command without extra input received